apps/api/src/services/sendgrid_handler.py

```diff
@@ -0,0 +1,110 @@
+# using SendGrid's Python Library
+# https://github.com/sendgrid/sendgrid-python
+import os
+from enum import Enum
+from logging import getLogger
+from typing import Iterable, Literal, Tuple, TypedDict, Union, overload
+
+import aiosendgrid
+from httpx import HTTPStatusError
+from sendgrid.helpers.mail import Email, Mail, Personalization
+
+log = getLogger(__name__)
+
+SENDGRID_API_KEY = os.getenv("SENDGRID_API_KEY")
+
+
+class Template(str, Enum):
+    # TODO: provide actual template IDs
+    CONFIRMATION_EMAIL = "d-2026cde7bebd45ad85723443808c5817"
```

Comment:
Should these be stored in env variables or is it fine to leave it hard-coded for now?

Reply:
We can just leave them as is for now. Creating the templates is a one time thing, so if we do need to make any changes, we can simply update the template on SendGrid.